### DISCUSSION TITLE
Set Default Date Picker Time Only Not Working On SAFARI

### DISCUSSION THREAD

**Discussion by hci-ikahera** [Budibase/budibase#10129] [Help]
Hi everyone,
Anyone here know why set default value Date Picker Time Only does not work on Safari?
How I can fix this issue?
Thank you for any kind of help.

Regards,
Ika

**Comment by andz-bb** [comment]
Hi Ika

I was unable to recreate this on my end. I was using the latest version of Budibase Cloud (v.2.4.40) and versions 111 of Chrome and 16.2 of Safari for comparison.

<img width="1832" alt="image" src="https://user-images.githubusercontent.com/110921612/228558297-f22a4d85-d1ce-4027-a55a-dadbac7dcb78.png">

Could you share what version of Safari and OS you're using? Also could you specify which Handlebars or JS function you are using for the default value? I will do some testing on my end to see if it can be resolved for you. Thanks

**Reply by hci-ikahera** [reply]
So my case is I got a list of data from an API. Then when I select one of rows it will go to detail which looks like in the screenshots. I use Budibase v2.2.9. It works well on Chrome, but it doesn't on Safari 16.3

#Table List
<img width="834" alt="Screenshot 2023-03-30 at 14 36 20" src="https://user-images.githubusercontent.com/95213000/228767129-af311392-558a-45c6-930f-17e7b31f560d.png">

#Detail On Chrome
<img width="682" alt="Screenshot 2023-03-30 at 14 39 32" src="https://user-images.githubusercontent.com/95213000/228767155-00e6baf9-67b2-49ae-865a-e468b05cc64a.png">

#Detail On Safari
<img width="696" alt="Screenshot 2023-03-30 at 14 40 08" src="https://user-images.githubusercontent.com/95213000/228767181-45767164-8bed-41d7-be4f-766dbb3763a3.png">

**Reply by ConorWebb96** [reply]
Hey @hci-ikahera,

I've tried to replicate this behaviour you are seeing, it is showing the time for me in Safari.

I see you may have some sensitive information on your app. Because of this, would it be possible for you to create a dummy app where the issue is occurring and send us an app export of this so we can have a more in-depth look?

Also I noticed you are quite far behind, compared to the most up-to-date version of Budibase, could you try updating and trying this again. A fix for this may have been deployed, so its a good idea to try this to make sure!

Thanks!
